app/models/application_record.rb

```diff
@@ -21,7 +21,5 @@ def self.provlog
     @provlog ||= Logger.new Rails.root.join('log', 'prov.log')
   end
 
-  def provlog
-    self.class.provlog
-  end
+  delegate :provlog, to: self
```

Comment:
1. this feature is not enabled at the moment (see shared_configs for each env, as well as base settings here -- the setting is explicitly false in all of those places).
2. i tested that this would behave in the same way by starting and seeding my local instance, and running:
```ruby
preservation-catalog(dev)> MoabStorageRoot.take.provlog == ApplicationRecord.provlog
  MoabStorageRoot Load (2.0ms)  SELECT "moab_storage_roots".* FROM "moab_storage_roots" LIMIT $1  [["LIMIT", 1]]
=> true
preservation-catalog(dev)> 
```